plugins/modules/apt_rpm.py

```diff
@@ -14,7 +14,7 @@
 DOCUMENTATION = '''
 ---
 module: apt_rpm
-short_description: apt_rpm package manager
+short_description: Apt_rpm package manager
```

Comment:
According to Wikipedia (https://en.wikipedia.org/wiki/APT-RPM), the correct spelling of the package manager is APT-RPM:
```suggestion
short_description: APT-RPM package manager
```